dirsrvtests/tests/suites/schema/schema_reload_test.py

```diff
@@ -25,6 +28,143 @@
+    try:
+        with open(schema_filename, "w") as schema_file:
+            schema_file.write("dn: cn=schema\n")
+            schema_file.write("attributetypes: ( 8.9.10.11.12.13.14 NAME " +
+                              "'MoZiLLaaTTRiBuTe' SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 " +
+                              " X-ORIGIN 'Mozilla Dummy Schema' )\n")
+            schema_file.write("objectclasses: ( 1.2.3.4.5.6.7 NAME 'MozillaObject' " +
+                              "SUP top MUST ( objectclass $ cn ) MAY ( MoZiLLaaTTRiBuTe )" +
+                              " X-ORIGIN 'user defined' )')\n")
+
+    except OSError as e:
+        log.fatal("Failed to create schema file: " +
+                  "{} Error: {}".format(schema_filename, str(e)))
+
+
+    # run the schema reload task with the default schemadir
+    topo.standalone.tasks.schemaReload(schemadir=topo.standalone.schemadir,
+                                              args={TASK_WAIT: True})
+
+    subschema = topo.standalone.schema.get_subschema()
+    at_obj = subschema.get_obj(ldap.schema.AttributeType, 'MoZiLLaaTTRiBuTe')
+
+    assert at_obj is not None, "The attribute was not found on server"
+
+    log.info('Case 2: Duplicated schema except cases are not loaded.')
+
+    schema_filename = topo.standalone.schemadir + '/97test.ldif'
+    try:
+        with open(schema_filename, "w") as schema_file:
+            Mozattr1 = "MOZILLAATTRIBUTE"
+            schema_file.write("dn: cn=schema\n")
+            schema_file.write("attributetypes: ( 8.9.10.11.12.13.14 NAME " +
+                              "'MOZILLAATTRIBUTE' SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 " +
+                              "X-ORIGIN 'Mozilla Dummy Schema' )\n")
+            schema_file.write("objectclasses: ( 1.2.3.4.5.6.7 NAME 'MozillaObject' "+
+                              "SUP top MUST ( objectclass $ cn ) MAY ( MOZILLAATTRIBUTE ) "+
+                              "X-ORIGIN 'user defined' )')\n")
+
+    except OSError as e:
+        log.fatal("Failed to create schema file: " +
+                  "{} Error: {}".format(schema_filename, str(e)))
+
+    # run the schema reload task with the default schemadir
+    topo.standalone.tasks.schemaReload(schemadir=topo.standalone.schemadir,
+                                              args={TASK_WAIT: True})
+
+    subschema_duplicate = topo.standalone.schema.get_subschema()
+    at_obj_duplicate = subschema_duplicate.get_obj(ldap.schema.AttributeType, 'MOZILLAATTRIBUTE')
+
+    moz = re.findall('MOZILLAATTRIBUTE',str(at_obj_duplicate))
+    if moz:
+        log.error('case 2: MOZILLAATTRIBUTE is in the objectclasses list -- FAILURE')
+        assert False
+    else:
+        log.info('case 2: MOZILLAATTRIBUTE is not in the objectclasses list -- PASS')
+
+    Mozattr2 = "mozillaattribute"
+    log.info(f'Case 2-1: Use the custom schema with {Mozattr2}')
+    name = "test_user"
+    topo.standalone.add_s(Entry((f"cn={name},{DEFAULT_SUFFIX}", {
```

Comment:
Not able to use any ldapobject here, I tried using directly DSLdapObjects.create() but getting assertion error. Can do it using UnsafeExtensibleObjects but not sure about this.

Reply:
The issue is that you use 'Entry' object that will be deprecated one day - #3656 
I thing it's good enough if you'll just use direct add_s pyhton-ldap call:
https://www.python-ldap.org/en/python-ldap-3.3.0/reference/ldap.html#ldap.LDAPObject.add_s

Reply:
@droideck Hey I can change that, no issue but I am using Entry in checkTask() and it only accept that, any suggestion there?

Reply:
@droideck Hi I have made the changes let me know if you think that is correct or not. Thanks!